shortened dict

D-Maxwell opened the pull request:
No need to store each character as a string in a list, instead one may use a single string to achieve the same effect. One may also deem it easier to read.

D-Maxwell committed:
shortened dict

No need to store each character as a string in a list, instead one may use a single string to achieve the same effect.
One may also deem it easier to read.

peotr26 commented:
It doesn't work, have you tested it before making your pull request?

The output is random, sometimes it's slipping on two lines, sometimes it's adding blank spaces ; but I never get the output I'm supposed to get.

D-Maxwell committed:
Fixed

A better, modular way to recursively reference previous entries in the dictionary is definitely doable.

peotr26 commented:
Well now it's not as legible. I'm going to close this pull request. The way it was originally written makes it easy for almost anyone to understand what characters are going to be used.
